.storybook/components/DesignTokens/Tier2/TypographyUsage.jsx

```diff
@@ -1,89 +1,40 @@
 
 export class Tier2TypographyUsage extends Component {
   render() {
-    const renderTypeToken = (size, preset, legacySize) => {
-      // remove variant suffix
-      const normalizedPreset = preset.split('-')[0];
-      const { fontSize, lineHeight } = PRESET_SIZE_MAP[normalizedPreset];
-
-      const name =
-        `eds-theme-typography-${size}` + (legacySize ? `/ ${legacySize}` : '');
-
-      return (
-        <TokenSpecimen
-          comment={`Ref: preset-${preset}`}
-          name={name}
-          specimenClassName={styles[`typography-usage--${size}`]}
-          value={`${fontSize} / ${lineHeight}`}
-          variant="typography-body"
-        />
+    const values = {};
+    forEach(at(usages, 'eds.theme.typography')[0], (_, key) => {
+      values[key] = flatten(
+        at(usages, `eds.theme.typography.${key}`)[0],
+        `eds-theme-typography-${key}-`,
       );
-    };
-
-    return (
-      <div>
-        <Section title="Headline">
-          <Grid>
-            {renderTypeToken('headline-lg', '001-bold', 'h1')}
-            {renderTypeToken('headline-md', '002-bold', 'h2')}
-            {renderTypeToken('headline-sm', '003-bold', 'h3')}
-          </Grid>
-        </Section>
-
-        <Section title="Title">
-          <Grid>
-            {renderTypeToken('title-md', '004-bold', 'h4')}
-            {renderTypeToken('title-sm', '005-bold', 'h5')}
-            {renderTypeToken('title-xs', '007-bold')}
-          </Grid>
-        </Section>
-
-        <Section title="Label">
-          <Grid>
-            {renderTypeToken('label-lg-subtle', '005')}
-            {renderTypeToken('label-md-subtle', '007')}
-            {renderTypeToken('label-md', '007-bold')}
-            {renderTypeToken('label-sm', '009-bold')}
-          </Grid>
-        </Section>
-
-        <Section title="Body Text">
-          <Grid>
-            {renderTypeToken('body-lg', '004-light')}
-            {renderTypeToken('body-md', '005-light')}
-            {renderTypeToken('body-sm', '006-light')}
-            {renderTypeToken('body-xs', '008-light')}
-          </Grid>
-        </Section>
-
-        <Section title="Caption">
-          <Grid>
-            {renderTypeToken('caption-lg', '006-light')}
-            {renderTypeToken('caption-md', '008')}
-            {renderTypeToken('caption-sm', '010')}
-          </Grid>
-        </Section>
-
-        <Section title="Button">
-          <Grid>{renderTypeToken('button-label', '006-bold')}</Grid>
-        </Section>
-
-        <Section title="Overline">
+    });
+    const contents = Object.keys(values).map((section) => {
+      return (
+        <Section key={section} title={capitalize(section)}>
           <Grid>
-            {renderTypeToken('overline', '011')}
-            {renderTypeToken('overline-sm', '012')}
+            {Object.keys(values[section]).map((usage) => {
+              return (
+                <TokenSpecimen
+                  inlineStyles={{ font: values[section][usage] }}
```

Comment:
using inline styles avoids the need to map custom class names to the declaration